redmine/pom.xml~

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
```

Comment:
Please delete this file, it is called pom.xml~ and I believe it is the backup file which most text editors are producing.